samri/pipelines/fc.py

```diff
@@ -183,6 +183,7 @@ def seed_based_connectivity(ts, seed_mask,
 def correlation_matrix(ts,
 	brain_mask="~/ni_data/templates/DSURQEc_200micron_mask.nii.gz",
 	loud = False,
+	save_csv = True,
```

Comment:
`save_as` is better than `save_csv`. Ideally we allow the format to be determined from the string value wherever possible.